components/markets/MarketSearch.tsx

```diff
@@ -30,77 +33,97 @@ const MarketSearch = () => {
         <Search className="mr-4 text-ztg-blue" />
       </Link>
       <div className="hidden items-center lg:flex">
+        <input
+          ref={inputRef}
+          className={`h-10 transition-all ${
+            showSearch ? "max-w-[500px] px-3" : "max-w-[0px]"
+          } w-full  rounded-lg bg-sky-900  text-white outline-none`}
+          value={searchTerm}
+          placeholder="Search markets"
+          onChange={(event) => {
+            setShowResults(true);
+            setSearchTerm(event.target.value);
+          }}
+          onFocus={() => {
+            if (searchTerm?.length > 0) {
+              setShowResults(true);
+            }
+          }}
+        />
+        {showSearch && (
+          <button
```

Comment:
I think it would be helpful for the user to see on desktop screens that search is gathering results with a loading spinner too. Perhaps the search icon could be replaced with a loading spinner while that state is occuring?

 
![Screenshot 2023-11-29 at 11 54 32 AM](https://github.com/zeitgeistpm/ui/assets/14626415/30aa967f-a248-49be-adb6-5814ddb88d88)